Model/System/Config/Source/Product/Brand.php

```diff
@@ -3,18 +3,41 @@
 namespace Lipscore\RatingsReviews\Model\System\Config\Source\Product;
 
 use \Magento\Framework\Data\Collection;
+use Magento\Catalog\Model\Product;
+use Magento\Eav\Model\ResourceModel\Entity\Attribute\CollectionFactory as AttributeCollectionFactory;
+use Magento\Eav\Model\Entity\Type;
+use Lipscore\RatingsReviews\Model\Logger;
 
 class Brand implements \Magento\Framework\Option\ArrayInterface
 {
-    protected $eavConfig;
+    /**
+     * @var AttributeCollectionFactory
+     */
+    protected $attributeFactory;
+
+    /**
+     * @var Type
+     */
+    protected $type;
+
+    /**
+     * @var Logger
+     */
     protected $logger;
 
+    /**
+     * @param Logger $logger
+     * @param AttributeCollectionFactory $attributeFactory
+     * @param Type $type
+     */
     public function __construct(
-        \Lipscore\RatingsReviews\Model\Logger $logger,
-        \Magento\Eav\Model\Config $eavConfig
+        Logger $logger,
+        AttributeCollectionFactory $attributeFactory,
+        Type $type
```

Comment:
It's definitely shorter and more clear when we use `use` instead of full class name in the code.

But add use the full names in constructor dependency injections everywhere in the project. Please leave it as full names in the constructor instead of adding the, via `use`.

At the same time it's great you imported other classes via `use` and methods code is more clear now.

Reply:
Hey! I always use `use` but in this file all your classes had `full path` that this way I didn't change it. The same this logger. It is copied from old version. I did this PR using github API, but let me fix it. What important now it works with `select` and `int` type and it is the biggest benefit.